standalone_tool_manager.py

```
@@ -0,0 +1,296 @@
+import os
+import subprocess
+import logging
+from flask import Flask, render_template, request, jsonify
+from flask_socketio import SocketIO, emit
+import json
+import shutil
+
+# Configure logging
+logging.basicConfig(level=logging.INFO, format='%(asctime)s - %(levelname)s - %(message)s')
+
+class ToolManager:
```

Flake8 (7.2.0)

[error] expected 2 blank lines, found 1

(E302)